addons/knobs/src/components/Panel.js

```diff
@@ -169,33 +156,32 @@ export default class Panel extends React.Component {
 
     return (
       <PanelWrapper>
-        {groupIds.length > 0 && (
-          <GroupTabs
-            groups={groups}
-            onGroupSelect={this.onGroupSelect}
-            selectedGroup={this.state.groupId}
-          />
-        )}
-        <PanelInner>
+        {groupIds.length > 0 ? (
+          <TabsState panels={groups} />
+        ) : (
```

Comment:
The `PropForm` component went from always being rendered to conditionally being rendered, is that intended?

Reply:
Indeed, deliberate change